tracks/crystal/exercises/pangram/mentoring.md

```diff
@@ -0,0 +1,56 @@
+### Algorithm
```

Comment:
Please use the following as per the [README.md](https://github.com/exercism/website-copy/blob/master/README.md) of this repository:

```suggestion
### Reasonable solutions
```